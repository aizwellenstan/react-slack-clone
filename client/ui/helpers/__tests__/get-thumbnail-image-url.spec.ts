import { getThumbnailImageUrl } from 'client/ui/helpers/get-thumbnail-image-url';

test('get thumbnail image url', () => {
  expect(getThumbnailImageUrl('123')).toMatchSnapshot();
});
