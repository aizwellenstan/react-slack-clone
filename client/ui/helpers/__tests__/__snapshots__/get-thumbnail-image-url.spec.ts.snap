// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get thumbnail image url 1`] = `"https://img.youtube.com/vi/123/hqdefault.jpg"`;
